Task 2/Task 2.2/Task 2.2/Wall.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Task_2._2
+{
+    class Wall : Gameobj
+    {
+        public Wall(int x , int y, char skin ):base(Layers.Ground,x, y, false,skin)
```

Comment:
Wall can have different skin? I didn't see an imitation of a map generator that uses this. What if someone creates a wall that looks like player or floor?